king_phisher/plugins.py

```diff
@@ -189,20 +199,20 @@ def compatibility_iter(self):
 		if self._storage.get('minimum-python-version'):
 			# platform.python_version() cannot be used with StrictVersion because it returns letters in the version number.
 			available = StrictVersion(self._storage['minimum-python-version']) <= StrictVersion('.'.join(map(str, sys.version_info[:3])))
-			yield ('Minimum Python Version', self._storage['minimum-python-version'], available)
+			yield Requirement(_RequirementType.PYTHON_VERSION, self._storage['minimum-python-version'], available)
```

Comment:
Change the string here from `...self._storage['minimum-python-version']...` to `...self._storage[_RequirementType.PYTHON_VERSION.name]` to keep that string usage limited.